### PR TITLE
step description update

### PR DESCRIPTION
**Pull request by BanyikAnna**
### Checklist

- [ ] I've read and followed the [Contribution Guidelines](https://github.com/bitrise-steplib/.github/blob/main/CONTRIBUTING.md)
- [ ] `step.yml` and `README.md` is updated with the changes (if needed)

### Version

Requires a *MAJOR/MINOR/PATCH* [version update](https://semver.org/)

### CODE REVIEW

File: step.yml [hunk 1]
```diff
@@ -1,7 +1,34 @@
...
+
+  ### Configuring the Step
+  Before you start:
+  - Make sure you have connected your [Apple Service account to Bitrise](https://devcenter.bitrise.io/en/accounts/connecting-to-services/apple-services-connection.html).
+  Alternatively, you can upload certificates and profiles to Bitrise manually, then use the Certificate and Profile installer step before Xcode Archive
+  - Make sure certificates are uploaded to Bitrise's **Code Signing** tab. The right provisioning profiles are automatically downloaded from Apple as part of the automatic code signing process.
+
+  To configure the Step:
+  1. **Archive Path**: Specifies the archive that should be exported. The input value sets xcodebuild's `-archivePath` option.
+  2. **Select a product to distribute**: Describes which product to export.
```
Comment (godrei) [inline step.yml:14]: In case of the input, the value options are adding this context, but basically, here users need to `Decide if an App or an App Clip IPA should be exported`.

File: step.yml [hunk 2]
```diff
@@ -1,7 +1,34 @@
...
+  3. **Distribution method**: Describes how Xcode should export the archive.
```
Comment (godrei) [inline step.yml:15]: The value options would add more context here, I suggest something like: `Describes how Xcode should export the archive: development, app-store, ad-hoc, or enterprise.`

similarly to what we have for xcode-archive: https://github.com/bitrise-steplib/steps-xcode-archive/blob/master/step.yml#L22

File: step.yml [hunk 3]
```diff
@@ -1,7 +1,34 @@
 title: Export iOS and tvOS Xcode archive
 summary: Export iOS and tvOS IPA from an existing Xcode archive
 description: |-
-  Exports an IPA from an existing iOS and tvOS `.xcarchive` file.
+  Manages your code signing assets and exports an IPA from an existing iOS and tvOS `.xcarchive` file. You can add multiple **Export iOS and tvOS Xcode archive** Steps to your Workflows to create multiple different signed .ipa files.
```
Comment (godrei) [inline step.yml:4]: I would remove the first part `Manages your code signing assets and`, since currently this is an optional feature of the step and I think the first sentence should be about the step's main functionalities.

Automatic code signing could be mentioned after this section, similarly to xcode-archive:

```
Manages your code signing assets and exports an IPA from an existing iOS and tvOS `.xcarchive` file. You can add multiple **Export iOS and tvOS Xcode archive** Steps to your Workflows to create multiple different signed .ipa files.

With this Step, you can use automatic code signing in a [CI environment without having to use Xcode](https://developer.apple.com/documentation/xcode-release-notes/xcode-13-release-notes).
  In short, the Step:
  - Logs you into your Apple Developer account based on the [Apple service connection you provide on Bitrise](https://devcenter.bitrise.io/en/accounts/connecting-to-services/apple-services-connection.html).
  - Downloads any provisioning profiles needed for your project based on the **Distribution method**.
```

**Review by godrei — CHANGES_REQUESTED** [review]

File: step.yml (hunk 3 above)
Comment (BanyikAnna) [reply step.yml:4]: good point, thanks! :)

**Review by godrei — APPROVED** [review]
